manifests/kiali-community/1.31.0/kiali.v1.31.0.clusterserviceversion.yaml

```diff
@@ -493,6 +493,7 @@ spec:
           - post
         - apiGroups: ["extensions", "apps"]
           resources:
+          - daemonsets
```

Comment:
This will need to be reverted. We already released 1.31.0 and it did not have this permission. The 1.31.0 manifest should be locked down and not changed (I have to create the necessary PRs over in OLM land to publish 1.31.0 to operatorhub.io - that has not been done yet).

So you need to back out this line change (as well as the one in kiali-upstream). We then have to create a new set of manifest files and call it 1.32.0 (we have [a script](https://github.com/kiali/kiali-operator/blob/master/manifests/create-new-version.sh) to help do this) and then make the change there. If you want to wait until I'm back off PTO on Monday, I can do that.

```suggestion
```